rpcs3/Emu/Cell/Modules/cellNetCtl.cpp

```diff
@@ -283,15 +298,7 @@ error_code cellNetCtlNetStartDialogUnloadAsync(vm::ptr<CellNetCtlNetStartDialogR
 		return CELL_NET_CTL_ERROR_INVALID_SIZE;
 	}
 
-	if (g_cfg.net.net_status == CELL_NET_CTL_STATE_Disconnected)
-	{
-		result->result = CELL_NET_CTL_ERROR_NET_NOT_CONNECTED;
-	}
-	else
-	{
-		// Hack
-		result->result = CELL_NET_CTL_ERROR_DIALOG_CANCELED;
-	}
+	result->result = nph->get_net_status() == CELL_NET_CTL_STATE_IPObtained ? 0 : CELL_NET_CTL_ERROR_DIALOG_CANCELED;
```

Comment:
CELL_NET_CTL_STATE_IPObtained weird name

Reply:
Comes from PS3, not a personal choice.

Reply:
lol